site/src/components/NavigationBar/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as NavigationBar } from "./NavigationBar"
+export { default as Link } from "./components/Link"
+export { default as Menu } from "./components/Menu"
```

Comment:
The Kaizen site doesn't make use of `Menu` at all, so there's heaps of dead code that you can delete in this tree.

Reply:
I'd also recommend doing a sweep of props and seeing what else can be deleted, including CSS. E.g. It doesn't look like the Kaizen site nav uses badges at all, so we can remove everything related to those.

Reply:
Just did that, not able to completely remove badge CSS since the mobile menu is still using it, but it's now fixed to the default icon. Removed as much props as I can at first glance, without deep refactoring.